rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,95 @@
+# 0000: Name of RFC
+<!--^ Increment the RFC number in the file and in the filename. Take care not to conflict with other RFCs.--> 
```

Comment:
+1 to leaving it `0000` initially and having the team assign a number (perhaps the PR # of the initial strawperson PR for simplicity).

Reply:
Nitpick, but I would avoid associating an RFC with the initial PR ID for two reasons.

1. This means there will be huge gaps in RFC IDs
2. Multiple PRs will be required to get an RFC through all stages. So having the RFC ID associated with the first PR about it will likely be confusing

I would like RFC IDs to be pristine contiguous numbers, like IETF RFCs, Python PEPs and other such proposals mechanisms. Gaps in the numbers will simply represent RFCs that were abandoned.